tests/acceptance/test_mender-artifact.py

```diff
@@ -247,4 +247,10 @@ def test_artifacts_rootfs_size(self, versioned_mender_image, bitbake_path, bitba
         assert(match is not None)
         size_from_artifact = int(match.group(1))
         size_from_build = int(bitbake_variables["MENDER_CALC_ROOTFS_SIZE"]) * 1024
-        assert(size_from_artifact == size_from_build)
+        # some filesystems (eg. ubifs) may use compression or empty space may
+        # not be a part of the image, in which case the image will be smaller
+        # or equal to MENDER_CALC_ROOTFS_SIZE
+        assert size_from_artifact <= size_from_build
+        # assume that the difference will not be more than 30% of calculated
+        # size
+        assert size_from_artifact >= 0.7 * size_from_build
```

Comment:
We've actually had quite a number of "off-by-one" errors in this area, so I would argue that the exact match is pretty important for the filesystem types where it's possible to test this. Can we make this dependent on which filesystem type is being tested?